networks.json

```diff
@@ -320,5 +320,27 @@
         "url": "https://bigdipper.live/nym"
       }
     ]
+  },
+  {
+    "name": "Quicksilver",
+    "endpoint": "https://gql-testnet.quicksilver.forbole.com/v1/graphql",
+    "links": [
+      {
+        "name": "Testnet",
+        "chain_id": "innuendo-3",
+        "url": "https://bigdipper.live/quicksilver"
```

Comment:
I think the URL should be https://testnet.bigdipper.live/quicksilver for testnet?